elm/src/Main.elm

```diff
@@ -207,15 +170,47 @@ view model =
                 ]
 
 
-renderStop : Stop -> Html Msg
-renderStop stop =
-    Html.li []
-        [ Html.text (Debug.toString stop)
-        ]
+viewLadder : List Stop -> List Vehicle -> Html Msg
+viewLadder stops vehicles =
+    let
+        ladder =
+            Ladder.ladder stops vehicles
+    in
+    Html.div []
+        (List.map viewLadderRow ladder)
 
 
-renderVehicle : Vehicle -> Html Msg
-renderVehicle vehicle =
-    Html.li []
-        [ Html.text (Debug.toString vehicle)
+viewLadderRow : Ladder.LadderRow -> Html msg
+viewLadderRow row =
+    Html.div []
+        [ Html.text
+            (if row.leftTrain then
```

Comment:
I noticed this is an if/else unlike the logic for row.stopName, which is a case statement. Out of curiosity, would case not work here as well?

Reply:
Right now, `row.leftTrain` is a boolean, just showing whether there's a train or not. Using a case statement with cases `True` and `False` is not really good style.

In the future, we'll probably want to have more information in that field, so we can show more information than just "train or no train". When that happens, it will probably be a type that's better for `case`ing instead of `if`ing.